docs/src/content/docs/availablemodels.mdx

```diff
@@ -15,12 +15,16 @@ The Ultravox API currently provides the following models.
     </tr>
     <tr>
         <td class="font-mono">fixie-ai/ultravox-70B</td>
-        <td>70B variant of Ultravox. Supports tools.</td>
+        <td>Based on Llama 3.1 70B. Supports tools.</td>
     </tr>
     <tr>
         <td class="font-mono">fixie-ai/ultravox-8B</td>
-        <td>8B variant of Ultravox. Not recommended for most use cases. Tools are unlikely to work.</td>
+        <td>Based on Llama 3.1 8B. Not recommended for most use cases. Tools are unlikely to work.</td>
```

Comment:
I think the previous "8B variant of Ultravox" text is preferable because it gives us the freedom to e.g. try swapping in ToolACE.  (Same above.)